### PR TITLE
ci: Always run 'crc'-testsuite with `--log-level debug`

### PR DESCRIPTION
**Pull request by cfergeau**
Commits which were part of https://github.com/code-ready/crc/pull/2688

### CODE REVIEW

File: test/e2e/features/proxy.feature [hunk 1]
```diff
@@ -9,8 +9,8 @@ Feature: Behind proxy test
 
     Scenario: Start CRC
         Given executing crc setup command succeeds
-        And  executing "crc config set http-proxy http://192.168.130.1:3128" succeeds
-        Then executing "crc config set https-proxy http://192.168.130.1:3128" succeeds
+        And  executing crc config set http-proxy http://192.168.130.1:3128 command succeeds
```
Comment (gbraad) [inline test/e2e/features/proxy.feature:12]: I am not a big fan of removing the quote marks as it makes reading the test much harder... as you are leaving commands like `oc` intact.

Comment (cfergeau) [reply test/e2e/features/proxy.feature:12]: There currently is a mix of `And execute crc setup` and `And executing "crc setup"`, so this at least adds consistency.
I'm not sure it's possible to automatically add "--log-level debug" while keeping the quote marks. And readability/consistency is going to be quite bad if we decide to add it explicitly to each `And executing "crc --log-level debug setup"`

**Comment by cfergeau** [comment]
retest this now

**Comment by praveenkumar** [comment]
retest this please

**Comment by cfergeau** [comment]
retest this please

**Comment by cfergeau** [comment]
This is green, but PR is in flux, need to clean it up first
/hold

**Comment by cfergeau** [comment]
/retest
retest this please

**Comment by cfergeau** [comment]
retest this please

File: test/e2e/crcsuite/crcsuite.go [hunk 2]
```diff
@@ -368,7 +368,7 @@ func StartCRCWithDefaultBundleWithStopNetworkTimeSynchronizationSucceedsOrFails(
 	if !bundleEmbedded {
 		extraBundleArgs = fmt.Sprintf("-b %s", bundleLocation)
 	}
-	crcStart := crcCmd.CRC("start").WithDisableUpdateCheck().WithdisableNTP().ToString()
+	crcStart := crcCmd.CRC("start").WithDisableUpdateCheck().WithDisableNTP().ToString()
```
Comment (praveenkumar) [inline test/e2e/crcsuite/crcsuite.go]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**